backend/src/main/java/ch/puzzle/pcts/service/validation/ValidationBase.java

```diff
@@ -59,29 +68,31 @@ public void validateOnDelete(Long id) {
         throwExceptionWhenIdIsNull(id);
     }
 
-    private void throwExceptionWhenIdIsNull(Long id) {
+    public void throwExceptionWhenIdIsNull(Long id) {
```

Comment:
```suggestion
    private void throwExceptionWhenIdIsNull(Long id) {
```
This method can be private